.github/workflows/load-tests.yml

```diff
@@ -1,11 +1,9 @@
 name: load-tests
+
 on:
-  push:
-    branches: [main]
-    tags:
-      - "v[0-9]+.[0-9]+.[0-9]+*"
```

Comment:
IMO still useful to keep it upon new releases

Reply:
updated in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38483/commits/e2ae35ac6e3fc6861a7bdef9b110302e6e7b72fc